### ISSUE TITLE
[ENH] Imported YAML w/ incorrectly scoped keys should recc

### ISSUE THREAD

**Issue by jdunhamvrtx** [rancher/rancher#27810]
tl;dr - Importing yaml with error(s) prompts a huge red popup with k8s error codes. This could be formatted with helpful tips and links to relevant k8s docs.

**What kind of request is this (question/bug/enhancement/feature request):**
Enhancement

**Steps to reproduce (least amount of steps as possible):**
Import YAML with a yaml key at the wrong scope.

**Result:**
Rancher will pop up an error dialog with K8s error response

**Desired / Helpful Behavior:**
There is a key with array in the response for this error type, e.g.,

`error validating data: [ValidationError(StatefulSet.spec.template): unknown field "securityContext" in io.k8s.api.core.v1.PodTemplateSpec, ...]`

To make this friendlier we could,
- format this error paragraph so that each `ValidationError:` is a new line,
- provide a hint on the solution by searching the API for the field; say where it should go, say it's not found, say it's wrong API version, etc.
- provide a series of URLs to the Spec docs, i.e. Statefulset, securityContext
- provide a checkbox to load up the Rancher UI of the offending API type with the options specified.

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
2.4
- Installation option (single install/HA):
Not relevant

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported):
Custom
- Machine type (cloud/VM/metal) and specifications (CPU/memory):
VM
- Kubernetes version (use `kubectl version`):
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.3", GitCommit:"2e7996e3e2712684bc73f0dec0200d64eec7fe40", GitTreeState:"clean", BuildDate:"2020-05-21T14:50:54Z", GoVersion:"go1.14.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.3+k3s1", GitCommit:"96653e8d86b74a1f73eadf09d4edc44778c2f8dd", GitTreeState:"clean", BuildDate:"2020-06-01T16:47:11Z", GoVersion:"go1.13.11", Compiler:"gc", Platform:"linux/amd64"}
